webapp/pages/admin/users.spec.js

```diff
@@ -0,0 +1,70 @@
+import { mount, createLocalVue } from '@vue/test-utils'
+import Users from './users.vue'
+import Styleguide from '@human-connection/styleguide'
+
+const localVue = createLocalVue()
+
+localVue.use(Styleguide)
+
+describe('Users', () => {
+  let wrapper
+  let Wrapper
+  let mocks
+
+  beforeEach(() => {
+    mocks = {
+      $t: jest.fn(),
+      $apollo: {
+        loading: false,
+      },
+    }
+  })
+
+  describe('mount', () => {
+    Wrapper = () => {
+      return mount(Users, {
+        mocks,
+        localVue,
+      })
+    }
+
+    it('renders', () => {
+      wrapper = Wrapper()
+      expect(wrapper.is('div')).toBe(true)
+    })
+
+    describe('search', () => {
+      let searchAction
+      beforeEach(() => {
+        searchAction = (wrapper, { query }) => {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-15T18:46:51Z" title="Monday, July 15th 2019, 8:46:51 pm +02:00">Jul 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,70 @@
+import { mount, createLocalVue } from '@vue/test-utils'
+import Users from './users.vue'
+import Styleguide from '@human-connection/styleguide'
+
+const localVue = createLocalVue()
+
+localVue.use(Styleguide)
+
+describe('Users', () => {
+  let wrapper
+  let Wrapper
+  let mocks
+
+  beforeEach(() => {
+    mocks = {
+      $t: jest.fn(),
+      $apollo: {
+        loading: false,
+      },
+    }
+  })
+
+  describe('mount', () => {
+    Wrapper = () => {
+      return mount(Users, {
+        mocks,
+        localVue,
+      })
+    }
+
+    it('renders', () => {
+      wrapper = Wrapper()
+      expect(wrapper.is('div')).toBe(true)
+    })
+
+    describe('search', () => {
+      let searchAction
+      beforeEach(() => {
+        searchAction = (wrapper, { query }) => {
```


I :heart: the naming convention here @roschaefer :smiley_cat:

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-15T20:57:38Z" title="Monday, July 15th 2019, 10:57:38 pm +02:00">Jul 15, 2019</time>_

---


`jenny` has no results, but `Jenny` should - the filters are case-sensitive unfortunately, in contrast to full text search, which indexes without cases.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-15T21:29:09Z" title="Monday, July 15th 2019, 11:29:09 pm +02:00">Jul 15, 2019</time>_

---


so the idea is to teach the admins this?